bundles/action/org.openhab.action.ciscospark/build.properties

```diff
@@ -0,0 +1,10 @@
+source.. = src/main/java/,\
+           src/main/resources/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               lib/ciscospark-client-1.0-SNAPSHOT.jar,\
```

Comment:
MIT license is OK.  It's good practice to add an about.html file explaining use of third-party content (like [this](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.chromecast/about.html)), but much of this repo didn't.